Kconfig

```diff
@@ -213,6 +213,7 @@ depends on FEATURE_RUNNABLE_PROGRAMS
 source "src/bin/sol-fbp-generator/Kconfig"
 source "src/bin/sol-fbp-runner/Kconfig"
 source "src/bin/sol-fbp-to-dot/Kconfig"
+source "src/bin/sol-tools/Kconfig"
```

Comment:
"sol-io-tools" and "one tools per directory" doesn't compute. Or you mean inside "sol-io-tools"? Anyway this bunch o single filed folders will just become noise. And my plan is to have a aggregator binary for the IO tools, like to run sol-gpio one could also run

sol-io gpio <gpio options>
sol-io aio <aio options>

And so on.